node/service/src/chain_spec.rs

```diff
@@ -1211,9 +1218,9 @@ pub fn versi_staging_testnet_config() -> Result<RococoChainSpec, String> {
 			TelemetryEndpoints::new(vec![(VERSI_STAGING_TELEMETRY_URL.to_string(), 0)])
 				.expect("Versi Staging telemetry url is valid; qed"),
 		),
-		Some(DEFAULT_PROTOCOL_ID),
-		None,
+		Some("versi"),
```

Comment:
nit: a constant would be nice!

Reply:
next time - I want this merged and out of the way :-)